2_process/src/process_data.R

```diff
@@ -0,0 +1,48 @@
+
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+
+
+# Process the data
+process_data <- function() {
```

Comment:
Once aspect of a targets-oriented pipelining approach that you'll get more familiar with later in this course is that we typically try to modularize our functions so that each function only produces a single output, whether that's a file or an object like a list or dataframe. So here, even though both the data processing and the data rendering steps do fall into the `2_process` phase, we'd actually split them into two separate functions, one which processes the data and saves it as a csv file, and a second that reads in that processed data, renders the diagnostics, and saves the diagnostics to a text file. Since your code runs fine as is, there's no need to make the changes now (unless you'd like to). In an upcoming issue you'll be reworking the code to fit into a `targets` framework, so will have a chance to modularize this more then.